databox/api/src/Controller/Admin/MetadataTagController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use Alchemy\MetadataManipulatorBundle\MetadataManipulator;
+use App\Entity\Core\MetadataTag;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Option\EA;
+use EasyCorp\Bundle\EasyAdminBundle\Context\AdminContext;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextField;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class MetadataTagController extends AbstractCrudController
+{
+    private array $knowTags;
+
+    public function __construct(MetadataManipulator $metadataManipulator)
+    {
+        $this->knowTags = array_keys($metadataManipulator->getKnownTagGroups());
```

Comment:
Should be filtered in autocomplete to prevent loading the whole list here.

Reply:
done